### ISSUE TITLE
Presentation Plan (03-24-21)

### ISSUE THREAD

**Issue by bagmk** [bagmk/Quantum_Machine_Learning_Express#4]
This is a possible topic we can discuss during the mid-report for the mentorship program. I will start to add possible figures here.

- Expressibility
- Relationship with Number of shot
- Convergence of ML

**Comment by Travis-S-IBM** [comment]
From 2021/03/26 meeting b/t @Travis-S-IBM and @bagmk.
Discussed slide organization

Slide 1: Motivation/background
"This project is a replication study of 2 papers on quantum circuits and machine learning".
Benefits to me: learn more about quantum circuits and near-term quantum machine learning
Benefit to community: show this work can be replicated using Qiskit only
Pictures from Tom's paper and Hannah's paper

Slide 2: Our work
"We have replicated results on circuit expressibility, and are making progress in replicating the machine learning work."
* Figure on expressibility
* Picture of a 4-qubit PQC (maybe circuit 1)
* Convergence plots for loss (1 data point & several data points)
* "Still figuring out how to achieve better convergence"
* "Wrote a simple implementation of the SPSA algorithm"
* "Coded up 19 different kinds of circuits using Qiskit"
* "Coded up the data sets using ideas from scikit-learn"

Slide 3: Next steps
"We will finish the replication -- including using classical neural networks -- and run the experiments on hardware."
* Conceptual figure about PQCs getting plugged into classical Neural networks
* "We plan to leverage code available in the Qiskit textbook to help us do this."
* Pull in Tom's figure as the thing we're trying to replicate
* ?
